src/index.css

```diff
@@ -1,5 +1,123 @@
 body {
+  background: #fff5fe;
   margin: 0;
   padding: 0;
   font-family: sans-serif;
 }
+
+@keyframes beat{
+	to { transform: scale(1.05); }
+}
+
+svg {
+  overflow: inherit !important;
+}
+
+#organ_5, #organ_10, #organ_11, #organ_17, #organ_16 {
```

Comment:
What is `#5`, `#10`, etc.? It's not clear. Can you not just give them all a `class`, to avoid multiple assignments to different `id`s?